x-pack/plugins/observability/public/hooks/use_values_list.ts

```diff
@@ -7,7 +7,7 @@
 
 import { capitalize, union } from 'lodash';
 import { useEffect, useState } from 'react';
-import { useDebounce } from 'react-use';
+import useDebounce from 'react-use/lib/useDebounce';
```

Comment:
This PR fixes an eslint rule that should have been firing, but wasn't. I've updated this import to comply with the fixed rule.